src/ContentScript/injectors/youtube.plugin.ts

```diff
@@ -1,89 +1,101 @@
   youtubeStyle!: HTMLElement;
 
-  failCounter!: number;
-
   insertStyle() {
     this.youtubeStyle = document.createElement('style');
     this.youtubeStyle.innerHTML = `
               #meta-contents {
                 display: flex;
                 flex-direction: row;
                 justify-content: space-between;
+                padding-top:40px;
               }
-              .button-container {
+              .button-container{
                 display: flex;
                 flex-direction: column;
                 justify-content: flex-end;
                 align-items: flex-end;
-                height: 100%;
-                width: 47%;
+                width: 605px;
                 margin-left: 20px;
-                text-align:left;
+                text-align: left;
                 border-left: 1px solid var(--yt-spec-10-percent-layer);
                 padding-left: 14px;
+                background: #eaf4f6;
+                border: 1px solid #eee;
+                padding: 11px;
+                box-shadow: rgb(0 0 0 / 24%) 0px 3px 8px;
+                border-radius: 22px;
+                box-sizing: content-box;
               }
-              .liker-tips {
-                text-align: left;
-                z-index: 10;
-              }
-              .liker-tips-title {
+              li:hover{
                 cursor: pointer;
-                -webkit-transition: opacity,color .2s ease-in-out;
-                transition: opacity,color .2s ease-in-out;
-                color: #28646e;
-                font-size: 16px;
-                line-height: 1.5em;
-                font-weight: 600;
+                text-decoration: underline;
               }
-              .liker-tips-title a {
-                color: #28646e;
+              .liker-button{
+                position: absolute;
+                top: 0;
+                left: 0;
+                width: 100%;
+                height: 100%;
               }
-              .liker-tips-content {
-                color: #4a4a4a;
-                font-size: 12px;
+              @media (max-width:1400px){
+                #meta-contents {
+                  display: flex;
+                  flex-direction: column;
+                  justify-content: space-between;
+                }
+                .button-container{
+                  display: flex;
+                  flex-direction: row;
+                  justify-content: flex-end;
+                  align-items: flex-end;
+                  width: 700px;
+                  margin-left: 20px;
+                  text-align: left;
+                  border-left: 1px solid var(--yt-spec-10-percent-layer);
+                  padding-left: 14px;
+                  background: #eaf4f6;
+                  border: 1px solid #eee;
+                  padding: 15px;
+                  box-shadow: rgb(0 0 0 / 24%) 0px 3px 8px;
+                  border-radius: 22px;
+                  height: 181px;
+                }
```

Comment:
The following style is exactly same as default, so can be removed
```CSS
display: flex;
flex-direction: column;
justify-content: flex-end;
align-items: flex-end;
```

Maybe the button-container style can simplify into this

```CSS
.button-container {
  width: 605px;
  margin-left: 20px;
  padding: 11px;
  padding-left: 14px;
  background: #eaf4f6;
  border: 1px solid #eee;
  border-left: 1px solid var(--yt-spec-10-percent-layer);
  border-radius: 22px;
  box-shadow: rgb(0 0 0 / 24%) 0px 3px 8px;
  box-sizing: content-box;
  text-align: left;
}
@media (max-width: 1400px) {
  .button-container {
    display: flex;
    flex-direction: row;
    justify-content: flex-end;
    align-items: flex-end;
    width: 700px;
    margin-left: 20px;
    padding: 15px;
  }
}
```